src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -55,15 +55,15 @@ public class CodeStyleOptions
         public static readonly PerLanguageOption<CodeStyleOption<bool>> PreferIntrinsicPredefinedTypeKeywordInDeclaration = new PerLanguageOption<CodeStyleOption<bool>>(nameof(CodeStyleOptions), nameof(PreferIntrinsicPredefinedTypeKeywordInDeclaration), defaultValue: TrueWithNoneEnforcement,
             storageLocations: new OptionStorageLocation[]{
                 new EditorConfigStorageLocation("dotnet_style_predefined_type_for_locals_parameters_members"),
-                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferIntrinsicPredefinedTypeKeywordInDeclaration")});
+                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferIntrinsicPredefinedTypeKeywordInDeclaration.Dev15")});
```

Comment:
Given this would be used for any mythical post-Dev15 thing, perhaps call it it .CodeStyle or something as a more agnostic suffix?

(I'm getting visions of Common7 all over again.)